### DISCUSSION TITLE
Look & Feel Option default tab

### DISCUSSION THREAD

**Discussion by kay316** [pymedusa/Medusa#7500] [Ideas & Feature requests]
Hey there,

You know where you enable the look & feel option and you get Anime/Series Tabs? 

Can we please have the option to choose which tab is default? its having the Anime tab as default where i frequently use the series, 

![image](https://user-images.githubusercontent.com/6085298/70866334-49ed1700-1f79-11ea-90c7-cb0269bc2165.png)

![image](https://user-images.githubusercontent.com/6085298/70866320-317cfc80-1f79-11ea-89ed-43ba9809b6a3.png)

Thank you and Best Regards

**Comment by p0psicles** [comment]
You can change the order of the tabs.
